project-management.qmd

```diff
@@ -28,17 +28,20 @@ Our group uses the following hierarchy of meetings, from brief daily status repo
 
 ## Time Tracking
 
-Every individual in the group contributes to multiple projects.
+We use [Toggl](https://toggl.com) to track the time each group member spends working on each of the (many) projects that we all contribute to.
 
-With the launch of the Data Science Incubator, we found it prudent to track time allocated to each project.
-The goal is to be fair to our collaborators and funders by quantifying our time spent and making sure that we are fairly allocating our efforts across projects.
-This will also be used to budget for the time and costs of future projects.
+This began with the Data Science Incubator, when it became prudent to track the time spent working on each project.
+We now use it in order to be fair to our collaborators and funders by quantifying the time spent on projects, and making sure that we are allocating effort appropriately across projects.
+We also use it to populate the Group Effort Spreadsheet, which we use to communicate our activities to stakeholders within the university.
 
-The goal is not to account for 100% of each person's time.
-The goal is to be able to confidently and correctly report that we have spent a particular number of days or fraction of time on a project.
```

Comment:
I kind of liked this reminder that we are not expected to account for 100% of our time and that it is not being used to "make sure we are working" or anything like that

Reply:
Done via 5f8e43dec19a036f40fcfe51173d50b2284a1091